README.markdown

```diff
@@ -88,36 +84,86 @@ Storm/Mesos provides resource isolation between topologies. So you don't need to
 
 ## Mandatory configuration
 
-1. `mesos.executor.uri`: Once you fill in the configs and repack the distribution, you need to place the distribution somewhere where Mesos executors can find it. Typically this is on HDFS, and this config is the location of where you put the distibution.
+1. `mesos.executor.uri` or `mesos.container.docker.image`: Once you fill in the configs and repack the distribution, you need to place the distribution somewhere where Mesos executors can find it. Typically this is on HDFS, and this config is the location of where you put the distibution. Alternatively, you may use a Docker image in place of the executor URI. Take a look at the Dockerfile in the top-level of this repository for an example of how to use it.
 
 2. `mesos.master.url`: URL for the Mesos master.
 
 3. `storm.zookeeper.servers`: The location of the Zookeeper servers to be used by the Storm master.
 
-4. `nimbus.host`: The hostname of where you run Nimbus.
-
 ## Optional configuration
 
 * `mesos.supervisor.suicide.inactive.timeout.secs`: Seconds to wait before supervisor to suicides if supervisor has no task to run. Defaults to "120".
-* `mesos.master.failover.timeout.secs`: Framework failover timeout in second. Defaults to "3600".
+* `mesos.master.failover.timeout.secs`: Framework failover timeout in second. Defaults to "24*7*3600".
 * `mesos.allowed.hosts`: Allowed hosts to run topology, which takes hostname list as a white list.
 * `mesos.disallowed.hosts`: Disallowed hosts to run topology, which takes hostname list as a back list.
 * `mesos.framework.role`: Framework role to use. Defaults to "*".
 * `mesos.framework.checkpoint`: Enabled framework checkpoint or not. Defaults to false.
 * `mesos.offer.lru.cache.size`: LRU cache size. Defaults to "1000".
+* `mesos.offer.filter.seconds`: Number of seconds to filter unused Mesos offers. These offers may be revived by the framework when needed. Defaults to "120".
+* `mesos.offer.expiry.multiplier`: Offer expiry multiplier for `nimbus.monitor.freq.secs`. Defaults to "2.5".
 * `mesos.local.file.server.port`: Port for the local file server to bind to. Defaults to a random port.
 * `mesos.framework.name`: Framework name. Defaults to "Storm!!!".
 * `mesos.framework.principal`: Framework principal to use to register with Mesos
 * `mesos.framework.secret.file`:  Location of file that contains the principal's secret. Secret cannot end with a NL.
-* `supervisor.autostart.logviewer`: Default is true, if not false please add 128M to topology.mesos.executor.mem.mb
+* `mesos.prefer.reserved.resources`: Prefer reserved resources over unreserved (i.e., `"*"` role). Defaults to "true".
+* `supervisor.autostart.logviewer`: Default is true, if not false please add 128M to `topology.mesos.executor.mem.mb`.
```

Comment:
The default here is true, so it would make more sense for the default of `topology.mesos.executor.mem.mb` to include these 128 MB, and have the message here instead be "Default is true; if false, you may subtract 128 MB form `topology.mesos.executor.mem.mb`".